spyder/utils/tests/test_vcs.py

```diff
@@ -46,5 +46,46 @@ def test_git_revision():
     assert all([isinstance(x, str) for x in get_git_revision(root)])
 
 
+def test_get_git_refs():
+    output_branches = [
+        # Default working branches
+        'master', '3.x',
+        # Tags
+        'v1.0.0', 'v1.0.0beta1', 'v1.0.0beta10', 'v1.0.0beta2', 'v1.0.0beta3',
+        'v1.0.0beta4', 'v1.0.0beta5', 'v1.0.0beta6', 'v1.0.0beta7',
+        'v1.0.0beta8', 'v1.0.0beta9', 'v1.0.0rc1', 'v1.0.0rc2', 'v1.0.0rc3',
+        'v1.0.0rc4', 'v1.0.1', 'v1.0.2', 'v1.0.3', 'v1.1.0alpha1',
+        'v1.1.0alpha2', 'v1.1.0alpha3', 'v1.1.0alpha4', 'v1.1.0beta1',
+        'v1.1.0beta2', 'v1.1.0beta3', 'v1.1.0beta4', 'v1.1.0beta5',
+        'v1.1.0beta6', 'v1.1.0beta7', 'v1.1.0beta8', 'v1.1.0rc1', 'v2.0.0',
+        'v2.0.0alpha1', 'v2.0.0alpha2', 'v2.0.0alpha3', 'v2.0.0alpha4',
+        'v2.0.0alpha5', 'v2.0.0alpha6', 'v2.0.0beta1', 'v2.0.0beta2',
+        'v2.0.0beta3', 'v2.0.0beta4', 'v2.0.0beta5', 'v2.0.0beta6',
+        'v2.0.0beta6b', 'v2.0.0rc1', 'v2.0.0rc2', 'v2.0.0rc3', 'v2.0.1',
+        'v2.0.2', 'v2.0.3', 'v2.0.4', 'v2.0.5', 'v2.0.6', 'v2.0.7', 'v2.0.8',
+        'v2.1.0', 'v2.1.0alpha1', 'v2.1.0beta1', 'v2.1.0beta2', 'v2.1.0beta3',
+        'v2.1.0rc1', 'v2.1.0rc2', 'v2.1.1', 'v2.1.2', 'v2.1.3', 'v2.1.4',
+        'v2.1.5', 'v2.1.6', 'v2.1.7', 'v2.2.0', 'v2.2.0beta1', 'v2.2.0beta2',
+        'v2.2.0beta3', 'v2.2.0beta4', 'v2.2.0rc', 'v2.2.1', 'v2.2.2',
+        'v2.2.3', 'v2.2.4', 'v2.2.5', 'v2.3.0', 'v2.3.0beta1', 'v2.3.0beta2',
+        'v2.3.0beta3', 'v2.3.0beta4', 'v2.3.0dev1', 'v2.3.0dev2',
+        'v2.3.0dev5', 'v2.3.0dev6', 'v2.3.0rc', 'v2.3.1', 'v2.3.2', 'v2.3.3',
+        'v2.3.4', 'v2.3.5', 'v2.3.5.1', 'v2.3.5.2', 'v2.3.6', 'v2.3.7',
+        'v2.3.8', 'v2.3.9', 'v3.0.0', 'v3.0.0b1', 'v3.0.0b2', 'v3.0.0b3',
+        'v3.0.0b4', 'v3.0.0b5', 'v3.0.0b6', 'v3.0.0b7', 'v3.0.1', 'v3.0.2',
+        'v3.1.0', 'v3.1.1', 'v3.1.2', 'v3.1.3', 'v3.1.4', 'v3.2.0', 'v3.2.1',
+        'v3.2.2', 'v3.2.3', 'v3.2.4', 'v3.2.5', 'v3.2.6', 'v3.2.7', 'v3.2.8',
+        'v3.3.0',
+    ]
+    braches_tags, branch, files_modified = get_git_refs(__file__)
+    print(braches_tags)
+    print(branch)
+    print(files_modified)
```

Comment:
Please remove these prints.

Reply:
Why? they are very helpful when things fail on a test

Reply:
Ok, if you want to retain them, you need to add `# spyder: test-skip` next to them. Else this makes our tests fail. That was added by Rafa a long time ago to avoid spurious print's in our codebase.

Reply:
Hmmm I thought that check was ignored for `test_` files 🤔

Reply:
What about using logging instead, like we do in our actual code? [Pytest captures log messages](https://docs.pytest.org/en/latest/logging.html) and displays them just like ``stdout`` output.